sdk/src/main/scala/com/horizen/storage/SidechainHistoryStorage.scala

```diff
@@ -88,7 +88,17 @@ class SidechainHistoryStorage(storage: Storage, sidechainTransactionsCompanion:
 
   def blockById(blockId: ModifierId): Option[SidechainBlock] = {
     val blockIdBytes = new ByteArrayWrapper(idToBytes(blockId))
-    storage.get(blockIdBytes).asScala.flatMap(baw => sidechainBlockSerializer.parseBytesTry(baw.data).toOption)
+    val baw = storage.get(blockIdBytes).asScala
+    baw match {
+      case Some(value) => {
+        log.debug("blockById: serilized block : %s".format(BytesUtils.toHexString(value.data)))
```

Comment:
seems logs can be removed, since they wre used only for debugging.
I would suggest instead to add the log for `sidechainBlockSerializer.parseBytesTry(value.data)` Failure result case.
Like we do in other places:
```
def getWithdrawalEpochInfo: Option[WithdrawalEpochInfo] = {
    storage.get(withdrawalEpochInformationKey).asScala match {
      case Some(baw) =>
        WithdrawalEpochInfoSerializer.parseBytesTry(baw.data) match {
          case Success(withdrawalEpochInfo) => Option(withdrawalEpochInfo)
          case Failure(exception) =>
            log.error("Error while withdrawal epoch info information parsing.", exception)
            Option.empty
        }
      case _ => Option.empty
    }
  }
```